src/rockstor/storageadmin/fixtures/test_shares-services.json

```
@@ -129,8 +129,8 @@
     "model": "smart_manager.service",
     "pk": 17,
     "fields": {
-        "name": "ztask-daemon",
-        "display_name": "ZTaskd",
+        "name": "rockstor-scheduling",
+        "display_name": "Rockstor Scheduling",
```

```suggestion
        "name": "scheduling",
        "display_name": "Scheduling",
```

@Hooverdan96 Although it has no effect on the associated test outcomes, best we change this to what is actually now used - as per prep_db.py. Ideally this should also contain our added services - but there are as-yet also un-referenced by the associated tests.